scripts/More_Options_to_Test/show_config.txt

```diff
@@ -829,6 +829,9 @@ indent_paren_close              Unsigned Number
   1: Align under the open paren
   2: Indent to the brace level
 
+indent_paren_after_func_def     { False, True }
+  Controls the indent of the open paren after a function definition, if on it's own line.If True, indents the open paren
```

Comment:
> __after__ a function definition

The parenthesis that are indented in your example are part of a function definition/declaration.